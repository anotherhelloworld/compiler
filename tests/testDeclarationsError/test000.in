const a = 10;
var
    b : integer = 15;
arr : array[a..b] of integer;