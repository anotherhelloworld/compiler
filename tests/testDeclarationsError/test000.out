Const expression expected Line 4 Column 17
